frontend/src/Pages/AuthenticationPage/ChangePasswordFirstTimePage.jsx

```diff
@@ -60,10 +60,14 @@ export function ChangePasswordFirstTimePage() {
           label="New Password"
           name="newPassword"
           rules={[
-            {
-              required: true,
-              message: PASSWORD_REQUIRED
-            },
+            ({ getFieldValue }) => ({
+              validator() {
+                if (getFieldValue("newPassword") !== "") {
```

Comment:
I saw you have used getFieldValue("abc") !== "" syntax somewhere. You should extract as a static method then resued where you need